src/client/pages/ConsentsFollowUp.stories.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable functional/immutable-data */
+import React from 'react';
+import { Meta } from '@storybook/react';
+
+import { ConsentsFollowUp } from './ConsentsFollowUp';
+
+export default {
+  title: 'Pages/ConsentsFollowUp',
+  component: ConsentsFollowUp,
+} as Meta;
+
+export const Defaults = () => (
+  <ConsentsFollowUp newsletters={[]} consents={[]} />
+);
+Defaults.story = {
+  name: 'with no entities',
+};
+
+export const NewsLetters = () => (
+  <ConsentsFollowUp
+    newsletters={[
+      {
+        id: '0',
+        nameId: 'n0',
+        description: 'Newsletter description',
+        name: 'Newsletter Name',
+      },
+    ]}
+    consents={[]}
+  />
+);
+NewsLetters.story = {
+  name: 'with a single newsletter',
+};
+
+export const MultipleNewsLetters = () => (
+  <ConsentsFollowUp
+    newsletters={[
+      {
+        id: '0',
+        nameId: 'n0',
+        description: 'Newsletter description',
+        name: 'Newsletter Name',
+      },
+      {
+        id: '1',
+        nameId: 'n1',
+        description: 'Another newsletter description',
+        name: 'Another Newsletter Name',
+      },
+    ]}
+    consents={[]}
+  />
+);
+MultipleNewsLetters.story = {
+  name: 'with multiple newsletters',
+};
```

Comment:
ah this is why in standup you mentioned why this was an array, yeah i guess we should refactor this to only take a single newsletter or use one of the particular newsletters in this array